packages/flutter_tools/lib/src/reporting/crash_reporting.dart

```diff
@@ -26,15 +26,61 @@ const String _kStackTraceFileField = 'DartError';
 /// it must be supplied in the request.
 const String _kStackTraceFilename = 'stacktrace_file';
 
+class CrashDetails {
+  CrashDetails({
+    @required this.command,
+    @required this.error,
+    @required this.stackTrace,
+    @required this.doctorText,
+  });
+
+  String command;
+  dynamic error;
+  StackTrace stackTrace;
+  String doctorText;
+}
+
+/// Reports information about the crash to the user.
+class CrashReporter {
```

Comment:
I think this needs tests.

Reply:
I've added a simple test that verifies that `CrashReporter.informUser` prints some key strings.  Did you have anything beyond that in mind?

Reply:
Good enough, thanks!